src/sentry/testutils/cases.py

```diff
@@ -1186,6 +1187,60 @@ def _send_buckets(cls, buckets, entity):
         )
 
 
+class SessionMetricsReleaseHealthTestCase(SessionMetricsTestCase):
```

Comment:
This class is a duplication of lines 1130 to 1168, can we figure out a way to pass use case without duplicating the code?